lib/presentation/widget/destination_picker/destination_picker_viewmodel.dart

```diff
@@ -47,48 +48,61 @@ class DestinationPickerViewModel extends BaseViewModel {
   final GetAllSharedSpacesInteractor _getAllSharedSpacesInteractor;
   final AppNavigation _appNavigation;
   final BehaviorSubject<WorkGroupNodesSurfingArguments> currentNodeObservable = BehaviorSubject.seeded(null);
-  final uploadDestinationTypeList = [UploadDestinationType.mySpace, UploadDestinationType.workGroup];
+  final destinationTypeList = [DestinationType.mySpace, DestinationType.workGroup];
```

Comment:
`destinationTypeList` can be initialized with shorter way like this: `DestinationType.values`